<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DesuraTools</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>desuratools_256.png</normaloff>desuratools_256.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="iconSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="accountNameContainer">
      <item>
       <widget class="QLabel" name="desuraAccountName_label">
        <property name="text">
         <string>Desura Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="desuraAccountName_input">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="placeholderText">
         <string>Your Desura Username</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="desuraAccountName_verify">
        <property name="text">
         <string>Load Games</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="steamID_label">
        <property name="text">
         <string>Steam ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="steamID_input">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="contextMenuPolicy">
         <enum>Qt::DefaultContextMenu</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="installedGames_tab">
         <attribute name="title">
          <string>Installed Desura Games</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QListWidget" name="installedGames_list">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="ownedGamesTab">
         <attribute name="title">
          <string>Owned Desura Games</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QListWidget" name="ownedGames_list">
            <property name="contextMenuPolicy">
             <enum>Qt::CustomContextMenu</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="gameInfoContainer">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="gameIcon_label">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="gameName_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="gameShortName_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="selectAllButton">
        <property name="text">
         <string>Select All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh List</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="installButton">
        <property name="text">
         <string>Install Selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="generateDesuraReport_action"/>
    <addaction name="refreshLists_action"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="generateDesuraReport_action">
   <property name="text">
    <string>Generate Desura Report</string>
   </property>
  </action>
  <action name="refreshLists_action">
   <property name="text">
    <string>Refresh All Games</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>desuraAccountName_input</sender>
   <signal>returnPressed()</signal>
   <receiver>desuraAccountName_verify</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
